jobs/registry/templates/config/registry.conf

```diff
@@ -0,0 +1,28 @@
+---
+version: 0.1
```

Comment:
That is the version of the configuration itself, apparently so that docker registry can do something fancy with partial reconfiguration?  The Registry docs aren't to explicit in this regard (https://docs.docker.com/registry/configuration/#version)